mappings/net/minecraft/world/dimension/OverworldDimension.mapping

```diff
@@ -2,3 +2,6 @@ CLASS btk net/minecraft/world/dimension/OverworldDimension
 	METHOD <init> (Lbcq;Lbti;)V
 		ARG 1 world
 		ARG 2 type
+	METHOD b (FF)Lcmw;
+		ARG 1 arg1
```

Comment:
These names don't seem particularly useful

Reply:
Enigma does some strange things; I can't even see this method.

Reply:
Oh, huh. Gotta look into that.